docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,157 @@
+# KubeStellar Hub chart usage
+
+This documents explains how to use KubeStellar Hub chart to deploy a new instance of KubeStellar
+with a choise of user-defined Control Plains.
+
+The information provided is specific for the following release:
+
+```shell
+export KUBESTELLAR_VERSION=0.23.0-alpha.1
+```
+
+## Pre-requisites
+
+A list of executable that may be required during the deployment can be fund [here](./pre-reqs.md).
+At minimum [Helm](https://helm.sh/) is required.

```

Comment:
I do not understand what this remark about "at minimum" is trying to say. Aren't all the referenced requirements required?

Reply:
I will try to elaborate a bit. I guess what I was trying to say is that technically to just deploy the chart one only needs helm. Obviously to create a kind cluster one also need kind, to joing cluster one would need clusteradm.... but they are not strictly necessary to deploy the chart.